samples/WebApi/SampleWebApi.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RdXmlFile>rd.xml</RdXmlFile>
+    <RuntimeIdentifiers>win-x64</RuntimeIdentifiers> 
```

Comment:
Does this mean that the sample won't work out of the box on Unix? I do not think that we should have the RuntimeIdentifiers here.